packages/react-native-aztec/package.json

```diff
@@ -1,29 +1,42 @@
 {
   "name": "@wordpress/react-native-aztec",
   "version": "0.1.11",
+  "description": "Aztec view for react-native.",
   "private": true,
-  "license": "GPL-2.0",
+  "author": "The WordPress Contributors",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "wordpress",
+    "react-native"
+  ],
+  "homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/react-native-aztec/README.md",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/WordPress/gutenberg.git",
+    "directory": "packages/react-native-aztec"
+  },
+  "bugs": {
+    "url": "https://github.com/WordPress/gutenberg/issues"
+  },
+  "dependencies": {
+    "prop-types": "15.6.0"
+  },
+  "peerDependencies": {
+    "react": "*",
+    "react-native": "*"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
   "scripts": {
```

Comment:
Do you plan to run those commands directly from the package folder?

Reply:
I think those are only used to run the rn-aztec demo app. However, I don't think any of us use it at any point in our development right now. I'm not sure yet what we'll end up doing with it 🤔